api/src/main/java/org/apache/cloudstack/api/command/user/volume/ListVolumesCmd.java

```diff
@@ -136,6 +140,10 @@ public String getStorageId() {
         return storageId;
     }
 
+    public Boolean getForsystemvm() {
+        return forsystemvm;
```

Comment:
To prevent the NPE I would propose to change this for: `BooleanUtils.isTrue(forsystemvm)`